python/graphstorm/inference/ep_infer.py

```diff
@@ -140,11 +139,11 @@ def infer(self, loader, save_embed_path, save_prediction_path=None,
                 etype = infer_data.eval_etypes[0]
                 pred_shape = list(pred.shape)
                 pred_shape[0] = g.num_edges(etype)
-                pred_data = DistTensor(pred_shape, dtype=pred.dtype,
-                                       name='predict-'+'-'.join(etype),
-                                       part_policy=g.get_edge_partition_policy(etype),
-                                       # TODO: this makes the tensor persistent in memory.
-                                       persistent=True)
+                pred_data = create_dist_tensor(pred_shape, dtype=pred.dtype,
+                                               name='predict-'+'-'.join(etype),
+                                               part_policy=g.get_edge_partition_policy(etype),
+                                               # TODO: this makes the tensor persistent in memory.
+                                               persistent=True)
```

Comment:
Can we make the default value of persistent to True as in most of the cases, it is set to True.

Reply:
we may want to remove this argument.